Kitodo/src/test/java/org/kitodo/selenium/testframework/helper/WebDriverProvider.java

```diff
@@ -159,4 +168,32 @@ private static void extractTarFileToFolder(File file, File destinationFolder) {
         tarGZipUnArchiver.setSourceFile(file);
         tarGZipUnArchiver.extract("", destinationFolder);
     }
+
+    private static String fetchLatestStableChromeDriverVersion() {
+
+        String version = null;
+        try {
+
+            URL url = new URL(CHROME_DRIVER_LATEST_RELEASE_URL);
+            URLConnection urlConnection = url.openConnection();
+
+            BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(urlConnection.getInputStream()));
+
+            StringBuilder content = new StringBuilder();
+            content.append(bufferedReader.readLine());
+
+            bufferedReader.close();
+
+            version = content.toString();
+
+            logger.info("Latest Chrome Driver Release found: " + version);
+
+        } catch (MalformedURLException exception) {
+            logger.error("Failed to fetch latest Chrome Driver Release");
+        } catch (IOException exception) {
+            logger.error("Failed to fetch latest Chrome Driver Release");
+        }
+
+        return version;
```

Comment:
You are returning a `null` value in error case or after fetching the Exception. But this result is not checked and / or direct used later in method `provideChromeDriver`.

Reply:
There's no default value possible and if this part of the test suite crashes that's fine. There's no way for the tests to handle this scenario anyway so I think that's fine to possibly crash fast. The reason has been logged.

Reply:
I just run the tests with both `null` and `""` for the `version` and it doesn't make a difference. The tests output exceptions because there's no ChromeDriver to be instantiated due to the missing downloaded file. The test run does not crash but tries its best and fails.

I changed it to the empty string nonetheless since that's a better practice overall.